### PR TITLE
update for release

### PR DESCRIPTION
**Pull request by leej3**

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,6 @@ requirements:
   run:
     - python >3.6
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: It works with python 2 and python 3

File: recipe/meta.yaml [hunk 2]
```diff
@@ -22,7 +22,6 @@ requirements:
   run:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:23]: There are missing dependencies in host: setuptools and wheel. We should explicitly use them

File: recipe/meta.yaml [hunk 3]
```diff
@@ -22,7 +22,6 @@ requirements:
...
     - python >3.6
 
-
 test:
   imports:
     - smartypants
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:32]: It would be nice to add pip check

**Review by skupr-anaconda — COMMENTED** [review]
We do not would `noarch: python` but arch specific. So it would be better to remove `noarch: python` and build artifacts for win32/win64 platforms too.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**